api/src/main/java/org/openmrs/module/xforms/XformBuilder.java

```diff
@@ -529,6 +529,9 @@ public static void setNodeValue(Element node, String value) {
 		if (value == null)
 			value = "";
 		
+		if (node == null)
```

Comment:
Thinking about this again, i think throwing an illegal argument exception may be better than just silently ignoring it. What do you think? :)